src/Xamarin.Android.Build.Tasks/Tasks/CheckForInvalidResourceFileNames.cs

```diff
@@ -30,6 +30,10 @@ public override bool RunTask ()
 					resourceFile = resource.ItemSpec;
 				var fileName = Path.GetFileName (resourceFile);
 				var directory = Path.GetFileName (Path.GetDirectoryName (resourceFile));
+				char firstChar = fileName [0];
+				if (!Char.IsLetter (firstChar) && firstChar != '_') {
```

Comment:
[`char.IsLetter()`](https://docs.microsoft.com/en-us/dotnet/api/system.char.isletter?view=net-6.0#system-char-isletter(system-char)) determines whether or not a character is a "member of any category of Unicode letter".  This would include things such as `é` and `の`, which will *not* be supported by `aapt`.

This should instead use a range check:

```csharp
('a' <= firstChar && firstChar <= 'z') ||
('A' <= firstChar && firstChar <= 'Z') ||
firstChar == '_'
```